### DISCUSSION TITLE
How to use alternate themes?

### DISCUSSION THREAD

**Discussion by x4IPx** [SlRvb/Obsidian--ITS-Theme#33] [Q&A] [answered]
Hello !
In readme.md it is written that alternative themes get through Style Settings Plugin. But I do not understand how. I downloaded the plugin but did not find the opportunity to choose the topic you need.
Can you help me?   Very lucky "Tangerine Dunes")

**Comment by Borber** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/30563826/138376707-eef888ca-eb38-42f6-af39-75e17bddcd32.png)
Activate this snippet

![image](https://user-images.githubusercontent.com/30563826/138376980-797a5f02-211c-4076-85e7-542438ed2791.png)
Set in the plugin

**Comment by x4IPx** [comment]
Thank you very much!
